cluster/kubernetes/sync.go

```diff
@@ -206,6 +206,11 @@ func (c *Cluster) getAllowedResourcesBySelector(selector string) (map[string]*ku
 			return nil, err
 		}
 		for gv, e := range discErr.Groups {
+			if strings.HasSuffix(gv.Group, "metrics.k8s.io") {
+				// The Metrics API tends to be misconfigured, causing errors.
+				// We just ignore them, since it doesn't make sense to sync metrics anyways.
```

Comment:
@stefanprodan can you confirm whether this is true? I am not that familiar with the metrics API.

Reply:
GKE comes with the metrics server and it registers the `metrics.k8s.io` at bootstrap. For EKS and AKS where you would install the metrics server with Helm, I don't think it will be an issue and for GKE ignoring it will not cause any problems as far as I can tell.